frame/vault/src/tests.rs

```diff
@@ -142,6 +142,16 @@ prop_compose! {
 proptest! {
 	#![proptest_config(ProptestConfig::with_cases(10000))]
 
+	#[test]
+	fn liquidate_strategy_successfully_liquidates_a_strategy_account(
+		strategy_account_id in strategy_account(),
+		total_funds in valid_amounts_without_overflow_1()
+	) {
+		do_liquidate_strategy_successfully_liquidates_a_strategy_account(
+			strategy_account_id, total_funds
+		)
+	}
```

Comment:
Declarative macros effectively disable `rustfmt` and make interactive debugging harder, thus the reason why the core logic was moved to another function.